client/src/tasks.ts

```diff
@@ -51,6 +51,7 @@ export function buildDenoConfigTask(
   name: string,
   command: string | undefined,
   sourceUri?: vscode.Uri,
+  description: string = "",
```

Comment:
is description able to be null?  TaskRequestResponse has it string or null, this has it as only string.  Want to make sure we are consistent.